src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -70,8 +71,9 @@ const Sidebar = () => {
           }`}
           onClick={selectPrivateChatTab}
         >
-          {hasUnreadMessages && <div className="notification"></div>}
           <span>Messages</span>
+
+          {numberOfUnreadMessages > 0 && <>({numberOfUnreadMessages})</>}
```

Comment:
Is this going to have a space between `Messages` and the count?

eg. Is this going to render as `Messages(1)` or `Messages (1)`?

Reply:
Both options are fine for me

Reply:
Cool cool

---

@igachmovmelon My preference would be to have the space between them then, so: `Messages (1)` (assuming that looks ok in the UI once added/etc)